src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -610,7 +612,14 @@ public DatasetEditorProps getItemsEditorProps() {
 		DatasetEditorProps props =
 				DatasetEditorProps.create(
 						entityBundle.getEntity().getId(),
-						() -> closeItemsEditor(),
+						() -> {
+							ToastMessageOptions toastOptions = new ToastMessageOptions.Builder()
+									.setTitle("Dataset Saved")
+									.setPrimaryButton("Show Schema", () -> this.actionMenu.onAction(Action.SHOW_TABLE_SCHEMA))
+									.build();
+							DisplayUtils.notify("Edit the Dataset Schema to add additional annotation columns to this dataset", DisplayUtils.NotificationVariant.SUCCESS, toastOptions);
+							closeItemsEditor();
+						},

```

Comment:
Toast to display when saving a Dataset

Reply:
This is why I love user research, leading the user to a likely next logical step in the workflow.